bin/build_docker_images.sh

```diff
@@ -17,5 +12,9 @@ docker build --build-arg NODE_ENV=development \
              -t "${REPO_URL}/${IMAGE_NAME}:${DRIVE_VERSION}" \
              .
 
-docker push "${REPO_URL}/${IMAGE_NAME}:latest"
-docker push "${REPO_URL}/${IMAGE_NAME}:${DRIVE_VERSION}"
+# Login to Docker Hub
+echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
+
+# Push images to the registry
+docker push "${IMAGE_NAME}:latest"
+docker push "${IMAGE_NAME}:${VERSION}"
```

Comment:
`VERSION` var is not defined. I see `DRIVE_VERSION` which we could rename to `VERSION`.

Reply:
Good catch! I renamed to `VERSION` for consistency with other repos.